tests/test_wave_01.py

```diff
@@ -143,12 +143,13 @@ def test_moves_movie_from_watchlist_to_watched():
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
     
-    raise Exception("Test needs to be completed.")
+    #raise Exception("Test needs to be completed.")
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
-
-@pytest.mark.skip()
+    assert movie_to_watch in updated_data["watched"]
```

Comment:
This first assertion is good! The one on line 151 could use a second look. You're asserting that `movie_to_watch["title"]` (which is a string) is not in the list `updated_data["watchlist"]`. That assertion is correct however it's not testing what you want, i.e. that the `movie_to_watch` is not in the `updated_data["watchlist"]`.